lib/CXGN/Trial/TrialLayout/AbstractLayout.pm

```diff
@@ -505,6 +505,7 @@ sub retrieve_plot_info {
 
     my $accession_name = $accession->uniquename;
     my $accession_id = $accession->stock_id;
+    my $accession_type = $accession->type->name;
```

Comment:
For the source key in the layout, to prevent any downstream error, I used the original 'accession' key to indicate the source of the plot (accession, cross, family). Type of the plot source can be checked outside the layout.